### ISSUE TITLE
I'm looking for version 0.41.0

### ISSUE THREAD

**Issue by enroly-mike** [softprops/lambda-rust#56]

**Comment by brigand** [comment]
It seems the publish wasn't successful. The github action shows that step taking 0 seconds for whatever reason. The latest tags are :latest and :1.39.0 from 4 months ago. 

@softprops could you manually publish a new version or try the actions run again? I can't update my deps because they depend on recently stabilized features.

**Comment by softprops** [comment]
sry about that. I set up the project to push on tag `v0.2.6-rust-1.41.0` which should trigger a docker push to docker hub. Keep an eye out for [this workflow run to complete](https://github.com/softprops/lambda-rust/actions/runs/44092229)

**Comment by brigand** [comment]
The new tag is working as expected. Thanks softprops.

For anyone that comes across this, either of the following in serverless.yml will update to the newer tag.

```yaml
custom:
  rust:
    dockerTag: '0.2.6-rust-1.41.0'
```
```yaml
custom:
  rust:
    dockerTag: latest
```
